app/views/transcripts/_form.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if upload_new?(transcript) %>
+  <% if policy(transcript).create? %>
+    <%= render "transcripts/new", form: form, transcript: transcript %>
+  <% end %>
+<% elsif upload_stalled?(transcript) %>
+  <%= render "transcripts/error", form: form, transcript: transcript, delete_path: delete_path, retry_path: retry_path %>
```

Comment:
If I upload a non-allowed file (like a PNG) ... I'm getting stuck "processing forever", which eventually lands in this stalled/retry state.

That all is fine, but it looks like the issue is:

1. We set [the mime type](https://github.com/PRX/feeder.prx.org/blob/main/app/models/tasks/copy_transcript_task.rb#L36) to the bad image/png
2. We notice that's invalid, and set [status to invalid](https://github.com/PRX/feeder.prx.org/blob/main/app/models/tasks/copy_transcript_task.rb#L39)
3. BUT - unlike on the [Uncut model](https://github.com/PRX/feeder.prx.org/blob/main/app/models/uncut.rb#L2)... our validations are missing that "if status complete" bit.  So the validation still fails, and we can't save the record.

I think this validation just needs the `if:` status check copied over, so we skip it if status is invalid.